spec/support/test_helpers/find_api_helper.rb

```diff
@@ -22,7 +22,9 @@ def stub_find_api_provider_200(
     funding_type: 'fee',
     age_range_in_years: '4 to 8',
     vac_status: 'full_time_vacancies',
-    content_status: 'published'
+    content_status: 'published',
+    latitude: '51.498024',
+    longitude: '0.129919'
```

Comment:
```suggestion
    longitude: '0.129919',
```

Reply:
This is within a method call rather than JSON and so it doesn't accept the trailing comma.